src/sentry/web/frontend/base.py

```diff
@@ -233,6 +235,15 @@ def get_team_list(self, user, organization):
             with_projects=True,
         )
 
+    def create_audit_entry(self, request, **kwargs):
+        entry = AuditLogEntry.objects.create(
+            actor=request.user,
```

Comment:
This is unlikely to be an issue in practice, but worth noting that if `request.user` were `AnonymousUser`, this will error.